browser_extension/src/browser_action/browser_action.html

```diff
@@ -8,63 +8,75 @@
   <title>Chaddon</title>
   <link rel="stylesheet" href="/css/bootstrap/bootstrap-3.3.7.min.css">
   <link rel="stylesheet" href="/css/style.css">
+  <link rel="stylesheet" href="/css/Login-Form-Dark.css">
+  <link rel="stylesheet" href="/fonts/ionicons.min.css">
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Pacifico" />
+  <link href="/css/bootstrap-social.css" rel="stylesheet" >
+  <link rel="stylesheet" type="text/css" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" >
```

Comment:
You can remove this since we already include font awesome just before the closing `</body>` tag.

```html
  <script type="text/javascript" src="/js/fontawesome/fontawesome-all.js"></script>
```